core/src/main/resources/org/apache/spark/ui/static/executorspage.js

```diff
@@ -416,8 +422,7 @@ $(document).ready(function () {
                         },
                         {data: 'hostPort'},
                         {data: 'isActive', render: function (data, type, row) {
-                            if (row.isBlacklisted) return "Blacklisted";
-                            else return formatStatus (data, type);
+                                return formatStatus (data, type, row);
```

Comment:
weird indentation here?

Reply:
ok nevermind -- now I see the old code didn't seem to follow the convention in the rest of this file.  Not sure about the style guide for js, but I think this might be a bit cleaner if we put each element on its own line, so the nesting is cleaner, like it is here:

https://github.com/apache/spark/blob/master/core/src/main/resources/org/apache/spark/ui/static/executorspage.js#L459-L476

Reply:
Ok. Done.